src/assisted_test_infra/test_infra/controllers/node_controllers/node_controller.py

```diff
@@ -206,7 +206,7 @@ def notify_iso_ready(self) -> None:
     def set_dns(self, api_ip: str, ingress_ip: str) -> None:
         pass
 
-    def set_dns_for_user_managed_network(self) -> None:
+    def set_dns_for_none_platform(self) -> None:
```

Comment:
```suggestion
    def set_dns_for_none_platform(self) -> None:
```
Even though we will remove the user managed networking option from service's interface, I think it still makes sense to talk about "user managed networking" in the implementation, that means we require the controller to setup the network before cluster creation (required for none + oci platforms today).